trie/secure_trie.go

```diff
@@ -96,7 +96,7 @@ func (t *StateTrie) TryGetAccount(key []byte) (*types.StateAccount, error) {
 	var ret types.StateAccount
 	res, err := t.trie.TryGet(t.hashKey(key))
 	if err != nil {
-		log.Error(fmt.Sprintf("Unhandled trie error: %v", err))
+		log.Error("Unhandled trie error in StateTrie.TGA", "err", err)
```

Comment:
In this location, we pass the `err` on to the caller, so it's IMO not correct to call it unhandled.